### DISCUSSION TITLE
Solved: Anyway to set an SMTP sender name?

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#21132] [General]
Hey people,

My instance sends email using the email address. For example, the sending email is notification@domain.tld, people will get the email from "notification". see the following screenshot:
![image](https://user-images.githubusercontent.com/2678414/202870629-e5be5f8e-6e85-4808-b105-203bebe43c73.png)

Do we have some variable to set the sender user name to be something like site name? for example "Bassam Social"

**Comment by DavidGerhart** [comment]
Edit /home/mastodon/live/.env.production
Try adding:

SMTP_FROM_ADDRESS=username <notifications@{fqdomian}>

**Reply by ghost** [reply]
Thank you. This will only set the address as an email, not the name as a text. I'm looking to display a sender name as well.

**Reply by DavidGerhart** [reply]
Glad you worked it out.  It's the correct syntax for what you were asking.

**Comment by ghost** [comment]
In case you're not getting what I'm talking about, here is another screenshot:

![IMG_20221120_122146](https://user-images.githubusercontent.com/2678414/202897141-89adf9a2-b078-474d-a557-33557eb72169.jpg)

I'm using the email (smile@bassam.social) and I'd like for the email sender name to be (Bassam Social) Instead I get (smile).

I'd like to know if there is a variable that I can use to fix that and get the desired effect.

Thank you!

**Reply by ineffyble** [reply]
The previous reply was correct. SMTP_FROM_ADDRESS takes an email address in the format `Display Name <email@domain.com>` (the traditional format for email addresses that includes a sender name).

So you need to set

```
SMTP_FROM_ADDRESS=Bassam Social <smile@bassam.social>
```

**Reply by ghost** [reply]
Perfect!!
That did it. Thank you!
